autostart/light-locker/light-locker-pantheon.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
This is copied from a generated light-locker desktop file without translations.
I don't think they're important if we're using `NoDisplay`.